web/static/web/css/header.css

```diff
@@ -1,144 +1,332 @@
+    margin-right: var(--burger-margin);
     margin-bottom: -10px;
 }
 
-.burger:hover {
+#header #burger:hover {
     cursor: pointer;
 }
 
-.burger {
-    align-self: center;
+#header #burger i {
+    color: #FFFFFF;
+    font-size: 30px;
+}
+
+#header #nav {
     justify-self: center;
+    font-size: var(--font-size);
 }
 
-.burger i {
-    color: #FFFFFF;
-    font-size: 30px;
+#header #nav .nav-item,
+#header #side-nav .nav-item {
+    height: calc(var(--font-size) + 2em);
+    margin: calc(var(--nav-space-between) / 2);
+    padding: 0;
+    border-radius: 0;
+    justify-content: center;
 }
 
-.nav .ui.inverted.menu .item {
-    color: rgba(255, 255, 255, .7) !important;
+#header #nav .nav-item:hover,
+#header #side-nav .nav-item:hover,
+#nav-user-dropdown.active {
+    /* Disables property from Semantic UI */
+    background-color: transparent;
 }
 
-.lang-form {
-    display: none;
+#header #nav .nav-item .bubble-background,
+#header #side-nav .nav-item .bubble-background {
+    position: absolute;
+    width: 100%;
+    height: var(--font-size);
+    border-radius: 2em;
+    z-index: 1;
+    transition: width var(--dur) linear, height var(--dur) linear, background-color var(--dur) ease-out;
+}
+
+#header #nav .nav-item:hover .bubble-background,
+#header #side-nav .nav-item:hover .bubble-background {
+    width: calc(100% + var(--bubble-background-extra-width));
+    height: calc(var(--font-size) + 2em);
+    z-index: 2;
+}
+
+#header #nav .nav-item .nav-item-text,
+#header #side-nav .nav-item .nav-item-text {
+    z-index: 3;
+    transition: color var(--dur) ease;
+}
+
+#header #nav .nav-item:nth-child(1):hover .bubble-background {
+    background-color: rgba(239, 184, 14, 0.9);
+}
+
+#header #nav .nav-item:nth-child(2):hover .bubble-background {
+    background-color: rgba(0, 235, 179, 0.8);
+}
+
+#header #nav .nav-item:nth-child(3):hover .bubble-background {
+    background-color: rgba(12, 100, 141, 1); /* MAKE's blue primary color (12, 66, 106), with brightness turned up from 149 to 114 */
+}
+
+#header #nav .nav-item:nth-child(4):hover .bubble-background {
+    background-color: rgba(0, 235, 179, 0.8);
+}
+
+#header #nav .nav-item:nth-child(5):hover .bubble-background {
+    background-color: rgba(239, 184, 14, 0.9);
+}
+
+#header #side-nav .nav-item:nth-child(1):hover .bubble-background {
+    background-color: rgba(12, 100, 141, 1);
+}
+
+#header #side-nav {
+    --estimated-max-name-length: 10em;
+
+    margin: 0;
+    justify-self: end;
+    display: grid;
+    grid-template-columns: var(--estimated-max-name-length) calc(var(--logo-padding) + 10px);
+    justify-items: end;
+    font-size: var(--font-size);
 }
 
-@media screen and (max-width: 900px) {
-    .limg.large {
+#header #side-nav .nav-item {
+    margin-right: 0;
+}
+
+#header #side-nav #nav-user-dropdown .menu {
+    background-color: rgb(34, 43, 52);
+    border-radius: 7px;
+    /* For some reason doesn't work with `border: 2px rgb(220, 221, 222)` */
```

Comment:
You have to specify `border: 2px solid rgb(220, 221, 222)`

Reply:
Oh right, thanks! :)

Reply:
Might want to remove the comment as well